docs/topics/plugin-install.adoc

```diff
@@ -20,3 +21,18 @@ NOTE: This guide uses _Eclipse_ to refer to an installation of Eclipse or JBoss
 . Review the installation details and press *Next*.
 . Accept the terms of the license agreement and press *Finish* to install the plugin.
 . Restart Eclipse for the changes to take effect.
+
+.Install the Plugin for an Offline Environment
+
+. Download the link:{ProductDownloadURL}{EclipsePluginFilename}-{ProductVersion}.zip[{ProductShortName} tooling].
```

Comment:
Just wondering if "RHAMT tooling" is the right term for what they're actually downloading. Should it be more specific to the plugin?

Reply:
I've adjusted this to be the Eclipse Plugin Repository